suites/pacific/rbd/tier-1_rbd.yaml

```diff
@@ -19,6 +19,8 @@ tests:
       abort-on-fail: true
       module: install_prereq.py
       name: install ceph pre-requisites
+      config:
```

Comment:
Can we please review is_production usage here ?

Reply:
Removed